namui/sample/media/src/lib.rs

```diff
@@ -18,44 +18,46 @@ impl Component for MediaExample {
             ctx.state::<Option<MediaHandle>>(|| None);
         let (sliced_audio, set_sliced_audio) = ctx.state::<Option<FullLoadOnceAudio>>(|| None);
 
-        ctx.effect("load media", || {
+        ctx.async_effect("load media", (), move |_| async move {
             let mp3 = namui::system::media::new_media(
-                &namui::system::file::bundle::to_real_path("resources/audio.mp3").unwrap(),
+                namui::system::file::bundle::read("resources/audio.mp3")
+                    .await
+                    .unwrap(),
```

Comment:
No case for passing bundle path? I thought we talk about this on previous weekly meeting.

Reply:
Maybe later. I'm thinking of an interface to create media from the `ResourceLocation`, but I'm not sure for now.

Reply:
ok